com.microsoft.java.lsif.core/src/com/microsoft/java/lsif/core/internal/indexer/ASTUtil.java

```diff
@@ -45,6 +45,8 @@ public static CompilationUnit createAST(final ITypeRoot input, final IProgressMo
 			if (root[0] == null) {
 				final ASTParser parser = newASTParser();
 				parser.setSource(input);
+				parser.setResolveBindings(true);
+				parser.setBindingsRecovery(true);

```

Comment:
Any reason we need binding recovery here?

If an element can not be found, it makes more sense to me that we should simply skip it.